apps/codelab/src/app/codelabs/angular/material/material.component.html

```diff
@@ -21,10 +21,6 @@
 
 
 <slide-deck slideShortcuts slidesRouting slides-tracking>
```

Comment:
Are you just removing the buttons?

Reply:
Yeah, from here. Materail component is based on angular module which is loaded on FullLayout, which has all 4 buttons - such kind of logic)